### ISSUE TITLE
Show history of political party membership for elected representatives

### ISSUE THREAD

**Issue by kaerumy** [Sinar/sinar.representatives#72]
This is data which is supposedly stored in Malaysia's Popit Database along with with posts.

Include widget highlighting history specific to political party membership and posts.

Reported by: Faiz Mustafa @hexth

**Comment by kaerumy** [comment]
Not sure "traitor to democracy" should be a tag, political alliances change all the time, but we could possibly have criteria that flags/tags politicians that change parties too often.

Sometimes changing parties is a good thing. If a politician's current coalition/party is corrupt, is it a bad thing for them to switch?

One criteria that is definitely bad is when a politician runs on specific platform, but changes immediately upon election (possibly due to bribes, or position). Bribery is under blacklist criteria for corruption, and we can flag them for such.

Also maybe a key political highlights widget to remind people of such events.

**Comment by hexth** [comment]
Hi, @hexth here. I was suggesting the tag on twitter for those RECENTLY-ELECTED. Changing parties is a political norm and can be for good. But changing your parties, effectively betraying your electorate less than a week after being elected is definitely not something good. Change of alliance should be sorted before the election IMO. Not too sure for the case of whole party changing their alliances, as we dont typically see this often.

**Comment by kaerumy** [comment]
@hexth agree. I think this can work on large scale through linking politicians with some key political event.

* "5 Sabah reps switch alliances" 
    * Link to Rep 1
    * Link to Rep 2
    * ...

* Link to Sabah Event will also be shown on the rep page
